textdb/textdb-storage/src/main/java/edu/uci/ics/textdb/storage/LuceneDataStore.java

```diff
@@ -1,100 +1,15 @@
 package edu.uci.ics.textdb.storage;
 
-import java.io.IOException;
-import java.nio.file.Paths;
-import java.util.List;
-
-import org.apache.lucene.analysis.Analyzer;
-import org.apache.lucene.analysis.standard.StandardAnalyzer;
-import org.apache.lucene.document.Document;
-import org.apache.lucene.index.IndexWriter;
-import org.apache.lucene.index.IndexWriterConfig;
-import org.apache.lucene.store.Directory;
-import org.apache.lucene.store.FSDirectory;
-
-import edu.uci.ics.textdb.api.common.Attribute;
-import edu.uci.ics.textdb.api.common.FieldType;
-import edu.uci.ics.textdb.api.common.IField;
-import edu.uci.ics.textdb.api.common.ITuple;
-import edu.uci.ics.textdb.api.storage.IDataStore;
-import edu.uci.ics.textdb.common.exception.StorageException;
-import edu.uci.ics.textdb.common.utils.Utils;
-
-public class LuceneDataStore implements IDataStore{
+public class LuceneDataStore {
+    public static final String DATA_STORE_DIRECTORY = "../index";
+    private static int numDocuments = 0;

```

Comment:
why `static`? I think most likely we will have multiple pieces of `LuceneDataStore` for different type of documents.